.github/workflows/build-artifacts.yml

```diff
@@ -40,3 +40,9 @@ jobs:
       with:
         name: bin-wheel
         path: dist/bin-${{ env.BIN_VERSION }}-py3-none-any.whl
+
+    - name: Publish package 
+      uses: pypa/gh-action-pypi-publish@release/v1
+      with:
+        user: __token__
+        password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
The *pypi publish* should be only triggered on `release: [published]`

Reply:
Okay